kb/migrating-visibility-data-from-es6.md

```diff
@@ -0,0 +1,88 @@
+---
+slug: migrating-visibility-data-from-es6
+title: Migrating Visibility Data from ES6
+tags:
+  - kb-article
+date: 2022-29-08T00:00:00Z
+---
+
+Support for Elasticsearch v7 (ES7) was added to the 1.7.0. Temporal Server update.
+This introduces several breaking changes, including the removal of mapping types.
+These changes make ES6 incompatible with ES7.
+
+There are several ways to update your Temporal Server to support ES7.
+Upgrade ES6 to ES7, and then use one of the methods below to update your server.
+
+Visibility data can be migrated through reindexing.
+After updating Elasticsearch, run a reindexing process to move data from ES6 to ES7.
+
+:::note
+If you are using AWS Elasticsearch, only upgrade to version 7.7.
+:::
+
+## Fresh install
```

Comment:
```suggestion
## Fresh installation
```
Per [install](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/install). (Oddly, _invite_ is OK as a verb or a noun. Go fig.)